src/services/refactors/addOrRemoveBracesToArrowFunction.ts

```diff
@@ -55,12 +68,48 @@ namespace ts.refactor.addOrRemoveBracesToArrowFunction {
             body = needsParentheses(actualExpression) ? createParen(actualExpression) : actualExpression;
             suppressLeadingAndTrailingTrivia(body);
             copyLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+            copyTrailingAsLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+            // Copy the trailing comments after the return statement
+            // Add them to our trailingCommentsHolder to use later
+            copyTrailingComments(returnStatement, trailingCommentsHolder, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+
+
+            // If there are trailing comments
+            if (trailingCommentsHolder && trailingCommentsHolder.emitNode && trailingCommentsHolder.emitNode.trailingComments) {
+
+                // First we format our comment
+                // ex. " trailing comment " -> "/* trailing comment */"
+                const formattedComment = formatSynthesizedComment(trailingCommentsHolder.emitNode.trailingComments[0]);
+
+                // Add one space of padding to the comment
+                const paddedComment = ` ${formattedComment}`;
+
+                // TODO - add some type of check to know if plus 1 is needed
+                // Could be a function that is like getSemiColonPositionModifier, if true, return 1, else return 0
+                // creates a text change from end of function body, with length of comment, for the comment.
+                // create a new range from end of function body to end of return statement
+                // + 1 accounts for the semi-colon
+                // Add function to check if
+                // function hasSemiColon(/* check next token. If semi colon return true, else false */) {
+                //     return true
+                // }
+
+                newEdit = createTextChangeFromStartLength(func.body.end + 1, paddedComment.length, paddedComment);
+            }
         }
         else {
             Debug.fail("invalid action");
         }
 
-        const edits = textChanges.ChangeTracker.with(context, t => t.replaceNode(file, func.body, body));
+        const edits = textChanges.ChangeTracker.with(context, t => {
+            t.replaceNode(file, func.body, body);
+            // Check if we have a newEdit
+            if (newEdit) {
+                // Push the raw change into our array of textChanges
+                t.pushRaw(file, { fileName: file.fileName, textChanges: [newEdit] });

```

Comment:
The way they're doing it above, `t.replaceNode(file, func.body, body);` replaces a Node. But since our comment needs to go after the `body` token, we instead push a raw edit with our new comment.